workflows/rules/common.smk

```diff
@@ -26,10 +26,13 @@ METADATA_DIR = hf.get_metadata_dir()
 # OUTPUTS
 # -------
 # -- extended = list of the file names with a given wildcard
-AGGREGATE_DATA = hf.aggregate_output()
-CELL_COUNTS = hf.cell_count_output()
+CONVERTED_DATA = hf.parquet_output()
```

Comment:
`CONVERTED` is not specific. Can you think of a more explicit name? maybe `CYTOTABLE_OUTPUT_DATA`?

Reply:
Updated! I've also added documentation about the data level these variable names belong to.